sabnzbd/deobfuscate_filenames.py

```diff
@@ -240,40 +275,40 @@ def deobfuscate(nzo, filelist: List[str], usefulname: str):
         biggest_file = filelist[0]
     else:
         biggest_file = None
-
+    if not (biggest_file and os.path.isfile(biggest_file)):
+        # no file found
```

Comment:
No logging for this case?

Reply:
It's a fault case. I should never happen. I'll put in an ERROR logging.